scripts/pkg_in_pipe.py

```diff
@@ -67,24 +68,17 @@ def print_table_line(out, build, link, by):
 
 with open('report.html', 'w') as out:
     print_header(out)
-    for ver in ['8.2', '8.3']:
-        for pipe in [
-            'incoming',
-            'ci',
-            'testing',
-            'candidates',
-            'lab',
-        ]:
-            tag = f'v{ver}-{pipe}'
-            print_table_header(out, tag)
-            koji_output = check_output(['koji', 'list-tagged', '--quiet', tag]).decode('utf8')
-            builds = [line.split() for line in koji_output.splitlines()]
-            for build in builds:
-                koji_output = check_output(['koji', 'buildinfo', build[0]]).decode('utf8')
-                kv_lines = [line.split(': ') for line in koji_output.splitlines()]
-                build_info = dict(kv for kv in kv_lines if len(kv) == 2)
-                print(build_info)
-                id = re.search(r'\[(.+)\]', build_info['BUILD']).group(1)
-                print_table_line(out, build[0], f'https://koji.xcp-ng.org/buildinfo?buildID={id}', ' '.join(build[2:]))
-            print_table_footer(out)
+    tags = [f'v{v}-{p}' for v in ['8.2', '8.3'] for p in ['incoming', 'ci', 'testing', 'candidates', 'lab']]
+    for tag in tags:
```

Comment:
It does not seem necessary to precompute a list of tags.  This and the lack of specific details about what exactly gets parallelized makes reading the patch harder than necessary